Jenkinsfile

```diff
@@ -31,3 +42,37 @@ node('linux') {
 			sh 'make test'
 	}
 }
+
+def git_commit = ''
+def go_list = ''
+
+stage 'Go & Git Info'
+node('linux') {
+
+	def gopath = gopath_linux()
+	def workdir = workdir_linux(gopath)
+
+	dir(workdir) {
+		checkout scm
+
+		// HACK: Recommended approach for getting command output is writing to and then reading a file.
+		sh 'mkdir -p tmp'
+		sh 'go list . > tmp/GO_LIST'
```

Comment:
>otherwise, how exactly are we getting the SDK code in question?

In this PR, note the lines [here](https://github.com/vdice/controller-sdk-go/blob/410d82be404ff97f72ee956bbba46e3918585f70/Jenkinsfile#L77-L80) which pass along the actual PR commit sha and the actual PR go repo to the downstream `workflow-cli` job, which, as seen in the corresponding [PR](https://github.com/deis/workflow-cli/pull/128/files) takes those two values and swaps them in to the appropriate `version` and `package` values in its version of the `glide.yaml`.  It then runs `make glideup` to download that specific `controller-sdk-go` version and continues with the usual build, binary upload and e2e kickoff.

@arschles Does that make sense?  Are we missing any steps/details in this approach?